manifests/c/ChilliBits/Spice/0.14.3/ChilliBits.Spice.locale.en-US.yaml

```
@@ -0,0 +1,40 @@
+# Created with WinGet Releaser v2 using Komac v1.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ChilliBits.Spice
+PackageVersion: 0.14.3
+PackageLocale: en-US
+Publisher: ChilliBits
+PublisherUrl: https://chillibits.com
+PublisherSupportUrl: https://www.spicelang.com
+PrivacyUrl: https://www.chillibits.com/en/legal-notice
+Author: Marc Auberer
+PackageName: Spice
+PackageUrl: https://www.spicelang.com
+License: MIT
+LicenseUrl: https://raw.githubusercontent.com/spicelang/spice/main/LICENSE
+Copyright: (c) Marc Auberer 2021-2022

```

```suggestion
Copyright: Copyright (c) 2021-2023 ChilliBits
```
This is what it says over at the CopyrightUrl, do you want to update it in the manifest as well?